pkg/server/metrics/metrics.go

```diff
@@ -148,6 +154,48 @@ func newServerMetrics() *ServerMetrics {
 			"reason",
 		},
 	)
+	culledLeases := prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
is this metric the same as lease_delete_total with http_status_code 200?

Reply:
Pretty much, yeah.